tests/specs/2way/activeSpeaker.spec.ts

```diff
@@ -0,0 +1,92 @@
+import { logInfo } from '../../helpers/browserLogger';
+import { ensureThreeParticipants, toggleMuteAndCheck } from '../../helpers/participants';
+import Filmstrip from '../../pageobjects/Filmstrip';
+import Toolbar from '../../pageobjects/Toolbar';
+
+describe('ActiveSpeaker ', () => {
+    const context = {};
+
+    it('testActiveSpeaker', async () => {
+        await ensureThreeParticipants(context);
+
+        await toggleMuteAndCheck(context.p1, context.p2);
+        await toggleMuteAndCheck(context.p2, context.p1);
+        await toggleMuteAndCheck(context.p3, context.p1);
+
+        // participant1 becomes active speaker - check from participant2's perspective
+        await testActiveSpeaker(context.p1, context.p2, context.p3);
+
+        // participant3 becomes active speaker - check from participant2's perspective
+        await testActiveSpeaker(context.p3, context.p2, context.p1);
+
+        // participant2 becomes active speaker - check from participant1's perspective
+        await testActiveSpeaker(context.p2, context.p1, context.p3);
+
+        // check the displayed speakers, there should be only one speaker
+        await assertOneDominantSpeaker(context.p1);
+        await assertOneDominantSpeaker(context.p2);
+        await assertOneDominantSpeaker(context.p3);
+    });
+});
+
+/**
+ * Tries to make given participant an active speaker by un-muting it.
+ * Verifies from {@code participant2}'s perspective that the active speaker
+ * has been displayed on the large video area. Mutes him back.
+ *
+ * @param activeSpeaker <tt>Participant</tt> instance of the
+ * participant who will be tested as an active speaker.
+ * @param otherParticipant1 <tt>Participant</tt> of the participant who will
+ * be observing and verifying active speaker change.
+ * @param otherParticipant2 used only to print some debugging info
+ */
+async function testActiveSpeaker(
+        activeSpeaker: Paticipant, otherParticipant1: Participant, otherParticipant2: Participants) {
+    logInfo(activeSpeaker.driver, `Start testActiveSpeaker for participant: ${activeSpeaker.name}`);
+
+    const speakerEndpoint = await activeSpeaker.getEndpointId();
+
+    // just a debug print to go in logs
+    logInfo(activeSpeaker.driver, 'Unmuting in testActiveSpeaker');
+
+    // Unmute
+    await new Toolbar(activeSpeaker).clickAudioUnmuteButton();
```

Comment:
Rather than having to do `new Toolbar(participant).foo()`, how about: `partifipant.getToolbar().foo()` ?